parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java

```diff
@@ -670,26 +666,21 @@ public int queueSize() {
      */
     public int storedEventsCount() {
         ArrayList<Map<String, Object>> ar = this.getStoredQueue();
-        if (ar != null) {
-            return ar.size();
-        }
-        return 0;
+        return ar.size();
     }
 
     private class QueueManager extends AsyncTask<Void, Void, Void> {
         @Override
         protected Void doInBackground(Void... params) {
             ArrayList<Map<String, Object>> storedQueue = getStoredQueue();
             // if event queue is too big, push to persisted storage
-            if (eventQueue.size() >= queueSizeLimit + 1) {
+            if (eventQueue.size() >= QUEUE_SIZE_LIMIT + 1) {
```

Comment:
Suggestion (💡): This check is unnecessary complicated, consider changing it to:

```java
if (eventQueue.size() > QUEUE_SIZE_LIMIT) { ... }
```

PS: This is already the case with `if (storedEventsCount() > STORAGE_SIZE_LIMIT) { ... }` below. 🤷

Reply:
Nice, thanks!